examples/StandardFirmataWiFi/wifiConfig.h

```diff
@@ -140,11 +153,11 @@ char wep_key[] = "your_wep_key";
  * CONFIGURATION ERROR CHECK (don't change anything here)
  *============================================================================*/
 
-#if ((defined(ARDUINO_WIFI_SHIELD) && (defined(WIFI_101) || defined(HUZZAH_WIFI))) || (defined(WIFI_101) && defined(HUZZAH_WIFI))  || (defined(WIFI_101) && defined(ESP_WIFI))  || (defined(ESP_WIFI) && defined(HUZZAH_WIFI)) || (defined(ESP_WIFI) && defined(ARDUINO_WIFI_SHIELD)))
+#if ((defined(ARDUINO_WIFI_SHIELD) && (defined(WIFI_101) || defined(HUZZAH_WIFI))) || (defined(WIFI_101) && defined(HUZZAH_WIFI))  || (defined(WIFI_101) && defined(ESP8266_WIFI))  || (defined(ESP8266_WIFI) && defined(HUZZAH_WIFI)) || (defined(ESP8266_WIFI) && defined(ARDUINO_WIFI_SHIELD)))
```

Comment:
one hacky way to do it, would be for each Wi-Fi driver definition, increment a counter (`wifiDriverCount`). Initialize it to 0. If no drivers will included `wifiDriverCount = 0`, throw an error. If more than 1 driver is included (`wifiDriverCount > 1`), throw an error.